lsdcGui.py

```diff
@@ -4035,9 +4051,13 @@ def eraseDisplayCB(self): #use this for things like zoom change. This is not the
 
     def getCurrentFOV(self):
       fov = {"x":0.0,"y":0.0}
-      if (self.zoom2Radio.isChecked()):  #lowmagzoom      
-        fov["x"] = daq_utils.lowMagFOVx/2.0
-        fov["y"] = daq_utils.lowMagFOVy/2.0
+      if (self.zoom2Radio.isChecked()):  #lowmagzoom
+        if (daq_utils.sampleCameraCount==2):   #this is a hard assumption that when there are 2 cameras the second uses highmagfov   
+          fov["x"] = daq_utils.highMagFOVx
```

Comment:
this will need to be documented once we have that going - will be very easy to get tripped up. along with the camera URL names, etc. are all based on two low mag views and two high mag views.

the comment there is appropriate and useful.

worth thinking about how three cameras should be handled